cloudbridge/cloud/interfaces/services.py

```diff
@@ -965,7 +965,7 @@ def create(self, name, location=None):
         pass
 
 
-class BucketObjectService(PageableObjectMixin, CloudService):
+class BucketObjectService(CloudService):
```

Comment:
Shouldn't the bucket object service remain a pageable service?

Reply:
It is not iterable given that the list method needs a bucket parameter. The subservices are iterable.

Reply:
Ok makes sense...